interface/weno/facilities.php

```diff
@@ -44,37 +47,145 @@
             display: none;
         }
     </style>
+
+    <script>
+        function activateManagement(){
+            var pharm = document.getElementById('pharmacy');
+            var exists = pharm.classList.contains('hide');
+            if(exists){
+                //exist so yes
+                pharm.style.display= 'block';
+            }
+
+            var fac = document.getElementById('facility');
+            fac.style.display='none';
+        }
+        function activateFacility(){
+            var facility = document.getElementById('facility');
+            facility.style.display= 'block';
+            var management = document.getElementById('pharmacy');
+            management.style.display='none';
+        }
+        
+        function downloadPharmacies(){
+            $('#notch-pharm').removeClass("hide");
+            $('#pharm-btn').attr("disabled", true);
+            $.ajax({
+                url: "<? echo $GLOBALS['webroot']; ?>" + "/src/Rx/Weno/file_download.php",
+                type: "GET",
+                success: function (data) {
+                    $('#notch-pharm').addClass("hide");
+                    $('#pharm-btn').attr("disabled", false);
+                },
+                // Error handling
+                error: function (error) {
+                    $('#notch-pharm').addClass("hide");
+                    $('#pharm-btn').attr("disabled", false);
+                    console.log(`Error ${error}`);
+                }
+            });
+        }
+
+        function downloadPresLog(){
```

Comment:
these 2 functions look like they do exactly the same thing

Reply:
will resolve that.